### DISCUSSION TITLE
how to downgrade ssh-mitm

### DISCUSSION THREAD

**Discussion by osjekwknwjsk** [ssh-mitm/ssh-mitm#78] [Show and tell]
tell me please

**Comment by manfred-kaiser** [comment]
Why do you want downgrading ssh-mitm?

Or do you mean a downgrade attack?

**Reply by osjekwknwjsk** [reply]
yeah, i mean downgrade

**Reply by manfred-kaiser** [reply]
downgrading ssh-mitm is possible. see comment below.

I recommend using the pip method, it's the easiest and most reliable.

For other people reading this.

SSH-MITM doesnot support the downgrade attack. If you want to perform this attack sshmitm from the dsniff package (https://manpages.debian.org/unstable/dsniff/sshmitm.8.en.html)

SSH-MITM is for protocol version 2

sshmitm is for protocol version 1

**Comment by manfred-kaiser** [comment]
If there is an issue i can fix it.

**Reply by osjekwknwjsk** [reply]
no,its good but i just want to know how to downgrade, can you tell me the command ?

**Comment by manfred-kaiser** [comment]
> tell me please

Using pip: pip install ssh-mitm==VERSION

AppImage: download a specific release

Snap: https://unix.stackexchange.com/questions/394421/install-specific-previous-version-of-package-using-snap/448349#448349

**Reply by osjekwknwjsk** [reply]
thank you bro.
